### DISCUSSION TITLE
v8.1

### DISCUSSION THREAD

**Discussion by anothermartz** [anothermartz/Easy-Wav2Lip#12] [Announcements]
I decided to do a v8.1 release so that the small fixes I've done since releasing v8 are pushed to those using the Easy-Wav2Lip.bat

I've fixed some issues with the saving and deleting of the last detected face tracking data and added a new variable/tick box to force the deletion of it, useful if you're using 2 different files with the same name.

[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/anothermartz/Easy-Wav2Lip/blob/v8.1/Easy_Wav2Lip_v8.1.ipynb)

There is also a new [Easy-Wav2Lip_v8.1.bat](https://github.com/anothermartz/Easy-Wav2Lip/releases/download/v8.1_release/Easy-Wav2Lip_v8.1.bat) found here, so please use that especially if you're new to this tool as it has extra checks to make sure it installs properly. There are now more checks to make sure VSBuildTools is the 2022 version, that the system has an Nvidia card and that new enough drivers are installed which should ensure that Cuda 12 is installed.

<hr /><em>This discussion was created from the release <a href='https://github.com/anothermartz/Easy-Wav2Lip/releases/tag/v8.1_release'>v8.1</a>.</em>

**Comment by ratingtesting** [comment]
That is great! Thank you very much!!!
